deployment/cfn-asserts-lambda-layers.json

```diff
@@ -0,0 +1,34 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Asserts NodeJS Lambda Layer",
+    "Parameters": {
+        "LayerVersion": {
+            "Type": "String",
+            "Description": "Version of the layer",
+            "Default": "0.1.44",
+            "AllowedValues": [
+                "0.1.44"
+            ]
+        }
+    },
+    "Resources": {
+        "AssertsNodeJSAWSLambdaLayer": {
+            "Type": "AWS::Lambda::LayerVersion",
+            "Properties": {
+                "CompatibleRuntimes": [
+                    "nodejs14.x",
+                    "nodejs12.x"
```

Comment:
Should 13.x is also covered here? Figure if good with 12 and 14, then it is?

Reply:
AWS currently only supports Node.js 12 & 14 for Lambda runtimes 
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html?icmpid=docs_lambda_help